### DISCUSSION TITLE
Adding health information to game page (like seizure warning)

### DISCUSSION THREAD

**Discussion by riku55** [RetroAchievements/RAWeb#802] [Ideas]
I just got a comment on the game/set I deved (Maru's Mission) that it should have a seizure warning (and I agree that this game could need one). I think this might be a good thing. Putting a warning in the comments or the forum probably won't help much. AFAIK, Retroachievements doesn't have anything like this, but this might be a good thing to display visibly somewhere at the top of the game page. Worth considering, maybe for V2.

**Comment by HolyShinx** [comment]
It's really good idea, maybe include NSFW warning in it too?

**Comment by pinchanzo** [comment]
> 
> 
> It's a really good idea, maybe include NSFW warning in it too?

I 100% agree with this, this is absolutely a requirement in my eyes.

**Comment by televandalist** [comment]
I wouldn't go so far to deem it a requirement since RA is supplementary and doesn't actually supply the games. 

However, a lot of users discover certain games through RA, which has a number of minors in its userbase. An age gate on certain games wouldn't hurt and it's in the cards. 

But seeing as how the site is still condemned to V1, I suggest requesting a hub be made for specific health issues such as seizure warnings. The ulterior motive of the hubs is to collect metadata for when V2 arrives and there is an actual tagging/keyword system in place. I don't see how it couldn't be beneficial here. 

FYI there already is one for NSFW/Anime-Tiddies.

**Comment by jeffjenx** [comment]
Maybe a tagging system would be useful, where games can be tagged with multiple labels (and possibly label groups).

This would allow common tags to be reused and provide future flexibility, such as user preferences to exclude all tags tagged with Mature, Epilepsy, etc. or search and filter functionality.

**Comment by wescopeland** [comment]
https://github.com/RetroAchievements/RAWeb/pull/3990
